server/src/test/scala/services/graphstages/ArrivalsGraphStagePaxNosSpec.scala

```diff
@@ -0,0 +1,202 @@
+package services.graphstages
+
+import akka.testkit.{TestKit, TestProbe}
+import drt.shared.CrunchApi.MillisSinceEpoch
+import drt.shared.MilliTimes.oneDayMillis
+import drt.shared.Terminals.T1
+import drt.shared._
+import org.specs2.specification.AfterEach
+import services.arrivals.ArrivalDataSanitiser
+import services.crunch.CrunchTestLike
+import services.{PcpArrival, SDate}
+
+import scala.collection.mutable
+import scala.concurrent.duration._
+
+
+class ArrivalsGraphStagePaxNosSpec extends CrunchTestLike with AfterEach {
+  sequential
+  isolated
+
+  override def after: Unit = TestKit.shutdownActorSystem(system)
+
+  val nowForThisTest: SDateLike = SDate(2019, 10, 1, 16, 0)
+  
+  private def buildArrivalsGraphStage = new ArrivalsGraphStage(
+    "",
+    mutable.SortedMap[UniqueArrival, Arrival](),
+    mutable.SortedMap[UniqueArrival, Arrival](),
+    mutable.SortedMap[UniqueArrival, Arrival](),
+    mutable.SortedMap[UniqueArrival, Arrival](),
+    mutable.SortedMap[UniqueArrival, Arrival](),
+    pcpTimeCalc,
+    Set(T1),
+    ArrivalDataSanitiser(None, None),
+    oneDayMillis,
+    () => nowForThisTest
+  )
+
+  def pcpTimeCalc(a: Arrival): MilliDate = PcpArrival.pcpFrom(0, 0, _ => 0)(a)
+
+  "Given a live arrival with 0 pax that is not landing for more than 6 hours " +
+    "Then we should use the ACL pax numbers" >> {
+    val probe = TestProbe("arrivals")
+    val (aclSource, _, _, liveSource) = TestableArrivalsGraphStage(probe, buildArrivalsGraphStage).run
+
+    aclSource.offer(
+      List(arrival(actPax = Option(189), scheduledMillis = nowForThisTest.addHours(6).millisSinceEpoch))
+    )
+    liveSource.offer(
+      List(arrival(actPax = Option(0), scheduledMillis = nowForThisTest.addHours(6).millisSinceEpoch))
+    )
+
+    probe.fishForMessage(2 seconds) {
```

Comment:
I think we might get a false success here by catching the update from the acl feed first. We probably need another fishForMessage before the liveSource offer, and then we'd probably need to change another field in the live arrival to force another update if that makes sense